docs/current/architecture.md

```diff
@@ -22,68 +22,74 @@
+For simplicity, Vaquero is delivered as a single docker image with all of the required services available (server and agent). The same docker image can be used to start containers in both the control plane and remote agent sites.
 
-## Architecture
+![](https://ciscocloud.github.io/vaquero-docs/docs/current/jan17Arch.png)
+
+### Terminology
 
-Vaquero is delivered in one container for operational simplicity. Vaquero can run in multiple modes, including server, agent, and standalone (the combination of server and agent).
+The following terms are used throughout the documentation:
 
-![](jan17Arch.png)
+* `source of truth (SoT)`: all configuration necssary to define your infrastructure and available provisioning details (workflows, operating systems, boot configuration, unattended install scripts, and site/host-level information)
 
+### Vaquero Server
 
-#### `vaquero server`
+Vaquero server is the central service that maintains the configuration and workflows for provisioning your hardware. It is the service that users and Vaquero agents interact with. Vaquero server consists of the following components:
 
-The `vaquero` container in `server` mode implements the logic to receive updates from sources of truth and process the data model and make it a reality by providing information to agents. The vaquero server cluster leverages etcd to persist state, data models and configurations. Servers never call down to agents, they simply act as a distributed message queue for agents to fetch updates from, enabling scalable distributed servers.
+1. `User API (in progress)`: REST API for users to interact with the system and provide operational insights into booting hosts.
+1. `Updater`: Responsible for updating the data models when changes are detected to a SoT; changes could come from github webhooks or from local modifications.
+1. `Model API`: REST API that responds to agents looking to update their model cache; it also provides a state manifest that enables an agent to know what state its booting hosts are in.
+1. `Event API`: REST API that receives events from long running services on agents.
+1. `Server Controller`: Responsible for maintaining all services that Vaquero Server provides, while acting as an intermediary between the services.
+1. `State Engine`: The brains behind vaquero; understands the data model, looks at event and task history, and ultimately makes the decision on moving provisioning hosts from one state to the next.
+1. `Task Manager`: Responsible for running jobs on distributed task executors; e.g, running a container to execute a power management call to reboot a host for reprovisioning.
```

Comment:
numbers dude, the numbers

Reply:
It's a markdown thing, the numbers will be correct when it gets rendered.